feature/my/src/main/java/com/hankki/feature/my/myjogbodetail/MyJogboDetailRoute.kt

```diff
@@ -195,7 +195,7 @@ fun MyJogboDetailScreen(
                     title = jogboTitle,
                     chips = jogboChips,
                     userNickname = userNickname,
-                    shareJogbo = updateShareDialogState
+                    shareJogboDialog = updateShareDialogState
```

Comment:
State는 네이밍 맞추는게 어떨까용~
```suggestion
                    shareJogboDialogState = updateShareDialogState
```

Reply:
아 이거 전 PR에도 달렸는데...또..놓쳤네요..제가....